### PR TITLE
Update Cave Story 

### PR DESCRIPTION
**Pull request by lasers**

### CODE REVIEW

**Comment by RobLoach** [comment]
:+1:

**Comment by lasers** [comment]
I got boxart from http://www.cavestory.org/special-thanks.php
Got rid of !@#$% with gimp.
Copied logos from in-game snaps with gimp too.

The old boxart was not realistic.

File: Dinothawr/.ignore [hunk 1]
```diff
@@ -0,0 +1 @@
+# Placeholder
```
Comment (RobLoach) [inline Dinothawr/.ignore]: You'll want to revert adding this file.

Comment (lasers) [inline Dinothawr/.ignore]: This was an accident. Web interface sucks. :-)

**Comment by lasers** [comment]
I am not sure how I can do that under web interface before the merge.

**Comment by RobLoach** [comment]
Can close in favour of https://github.com/libretro/libretro-thumbnails/pull/172

**Pull request closed without merging**